packages/fxa-content-server/app/scripts/views/mixins/password-mixin.js

```diff
@@ -41,7 +41,6 @@ export default {
    */
   _updateShowPasswordLabel(passwordEls) {
     const $targetEl = this.$(passwordEls);
-    this._addShowPasswordLabel($targetEl);
```

Comment:
the issue here was the lazy creation of the label which changed the dom while the on-screen-keyboard had focus causing it to lose it's place on the input element. I switched to eager creation but set the style as 'empty'